bin/git-ranch

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+
+echo "Howdy pardner! Looks like you've mistaken a git command. Did you mean \`git branch\`? See \`git --help\`. Yee-haw!
+
+While yer in the ranchin' mood, why don't ya try this recipe for bacon 'n beans?
+
+Ingredients
+
+1 pound 90% lean ground beef*
```

Comment:
I, as a swede, feel left out when we use the imperial system. What do you think about adding a `--metric` and to convert this into kilograms?